FastStyleTransfer/Utility/Resize.swift

```diff
@@ -0,0 +1,24 @@
+import TensorFlow
+
+@differentiable(wrt: input, vjp: _vjpResizeNearestNeighbor(input:scale_factor:))
+public func resizeNearestNeighbor<Scalar: TensorFlowFloatingPoint>(_ input: Tensor<Scalar>, scale_factor: Float) -> Tensor<Scalar> {
```

Comment:
Use camel case for all names including argument labels.

```suggestion
public func resizeNearestNeighbor<Scalar: TensorFlowFloatingPoint>(_ input: Tensor<Scalar>, scaleFactor: Float) -> Tensor<Scalar> {
```

Reply:
Done